app/models/metric/helper.rb

```diff
@@ -198,4 +198,12 @@ def self.get_time_interval(obj, timestamp)
 
     start_time..timestamp
   end
+
+  def self.latest_metrics(resource_type, resource_ids = nil)
+    metrics = Metric.where(:resource_type => resource_type)
```

Comment:
@kbrock any way to make the syntax of the whole query (these 5 lines) nicer? Any syntactic sugar?